### ISSUE TITLE
Forms: Trusted person - missing tel/mail/send mail button

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#233]
```
See the french gouv definition here

    http://www.sante.gouv.fr/IMG/pdf/la_personne_de_confiance.pdf

```

Original issue reported on code.google.com by `eric.mae...@gmail.com` on 6 Dec 2012 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
started in the FullIdentity subform
```

Original comment by `eric.mae...@gmail.com` on 7 Dec 2012 at 8:03
- Changed title: **Forms: add Trusted person - missing City/State/ZipCode ; send mail button**
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric.mae...@gmail.com` on 20 Dec 2012 at 5:31
- Changed title: **Forms: Trusted person - missing tel/mail/send mail button**
- Changed state: **Accepted**
- Added labels: **Milestone-Release0.9.0**
- Removed labels: **Milestone-Release0.8.0**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision aa649b119f80.
```

Original comment by `eric.mae...@gmail.com` on 24 Jul 2014 at 9:13
- Changed state: **Fixed**
